### ISSUE TITLE
WebVTT file consists of a sequence of [text vs html] segments?

### ISSUE THREAD

**Issue by Webzeugmacher** [w3c/webvtt#443]
Minor clarification needed?

1.1 A simple caption file - in the first sentence after example of WebVTT-file
"You can see that a WebVTT file in general consists of a sequence of **text** segments"

Don't segments consist of a sequence of **html**?
While most segments in the example contain text-nodes only, at least one segment contains another node-type:

```
00:32.500 --> 00:33.500 align:left size:50%
<v Neil deGrasse Tyson><i>Laughs</i>
```

Thanks for the great work!!

**Comment by silviapfeiffer** [comment]
No, there's no html in a WebVTT file - only some markup that is similar to some HTML markup. We could say "sequence of possibly marked up text segments" instead, but this is the non-normative example section, so we decided to be brief.

**Comment by Webzeugmacher** [comment]
Ups, saw a known syntax and wanted to be too smart, it seems :)
Maybe link "text" to 6.4. WebVTT cue text parsing rules ?
Thanks for your time!

**Comment by silviapfeiffer** [comment]
Thanks for reading! Any further feedback welcome.

**Comment by gkatsev** [comment]
I think that extra text in this situation won't be helpful but maybe we should link to the WebVTT cue payload types section [4.2. Types of WebVTT cue payload](https://w3c.github.io/webvtt/#types-of-webvtt-cue-payload)? Or even specifically section [4.2.2. WebVTT caption or subtitle cue text](https://w3c.github.io/webvtt/#caption-text) as that section explains the possible spans that are allowed in caption text.
